server/mscalendar/availability.go

```diff
@@ -35,14 +35,14 @@ func (m *mscalendar) SyncStatusAll() (string, error) {
 		return "", err
 	}
 
-	index := userIndex.GetMattermostUserIDs()
-	mmIDs := []string{}
-	for _, id := range index {
+	allIDs := userIndex.GetMattermostUserIDs()
```

Comment:
_only_ if you do another push anyway... a nit of a nit, but maybe inline `userIndex.GetMattermostUserIDs()` and just use `ids` for the return? :)